src/App.scss

```diff
@@ -5,32 +5,3 @@ iframe {
 .App__title {
   text-align: center;
 }
-
-.PostInfo {
```

Comment:
The .PostInfo styles have been removed. Make sure to move them to a separate file, e.g. 'PostInfo.scss'